src/Components /Header/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Header.css";
+
+export default function Header() {
+    return (
+        <header className="header">
+            <span className="logo">
```

Comment:
If you don't plan on using this image you may as well nuke the span and img tags